dev/com.ibm.ws.microprofile.reactive.streams.operators.cdi/src/com/ibm/ws/microprofile/reactive/streams/cdi/ReactiveStreamsEngineProducer.java

```diff
@@ -23,7 +23,7 @@
 import javax.enterprise.inject.spi.InjectionPoint;
```

Comment:
Some unused imports? Also, should we use the smallrye version of this class ... somehow?